### DISCUSSION TITLE
Installation?

### DISCUSSION THREAD

**Discussion by iilis** [sandermvanvliet/RoadCaptain#68] [Q&A] [answered]
I downloaded the latest copy of the code.  How do I preform the initial installation?
I read through the ReadMe, but don't see how to actually install the app(s).
Thanks!

**Comment by iilis** [comment, marked as answer]
Never mind.  I had downloaded the source code zip before.  Had to download the MSI file to install.

**Comment by sandermvanvliet** [comment]
Glad that you found the right download :+1:

Seems like I need to update the documentation a bit with how to install. There is also some work to be done there on the features etc.
